protos/flyteidl/plugins/pytorch.proto

```diff
@@ -4,8 +4,22 @@ package flyteidl.plugins;
 
 option go_package = "github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/plugins";
 
+// Custom proto for torch elastic config for distributed training using 
+// https://github.com/kubeflow/training-operator/blob/master/pkg/apis/kubeflow.org/v1/pytorch_types.go
+message ElasticConfig {
+  string rdzv_backend = 1;
+  int32 min_replicas = 2;
```

Comment:
Do we just ignore workers if this is set?

Reply:
No, we still set it, see [here](https://github.com/flyteorg/flyteplugins/blob/b644427ef3af6ccb9b6771cdfa6191b2a6209e35/go/tasks/plugins/k8s/kfoperators/pytorch/pytorch.go#L97). In the kubeflow `PyTorchJob` the number of `replicas` is set outside of the `ElasticPolicy` (see [here](https://github.com/kubeflow/training-operator/blob/8a066f9386e9fb739815e9001f4fe53158227646/examples/pytorch/elastic/echo/echo.yaml#L13)) where we also set it for the non-elastic pytorch job.